lib/linter-views.coffee

```diff
@@ -153,9 +153,9 @@ class LinterViews
   _renderBubble: (message) ->
     bubble = document.createElement 'div'
     bubble.id = 'linter-inline'
-    bubble.appendChild Message.fromMessage(message)
+    bubble.appendChild Message.fromMessage(message, addPath: false, cloneNode: false)
```

Comment:
No need to specify `addPath`/`cloneNode` if those are false (the default), if you follow my previous advice on default :smile: